README.md

```diff
@@ -594,6 +594,11 @@ css(styles2.globals);
 
 It isn't determinate whether divs will be red or blue.
 
+## Minify class names
+
+Minify class names by setting the environment variable `process.env.APHRODITE_KEYS` 
```

Comment:
The reason I added another env variable is because it seems reasonable that someone might not want to minimize class names in production in order to make debugging production-only problems easier.